### PR TITLE
Change/remove backwards compatibility3

### PR DESCRIPTION
**Pull request by bartvanb**
Note that this was branched of the backwards-compatibility2 branch, so that can also be reviewed first

### CODE REVIEW

File: vantage6-common/vantage6/common/client/utils.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        A dictionary containing the secret and URI to generate the QR code
+    """
+    print("This server has obligatory two-factor authentication. Please scan "
+          "the QR code below with your favorite authenticator app (we "
+          "recommend the LastPass or Google Authenticator).")
+    print("After you have authenticated, please log in again.")
+    show_qr_code_image(json_data.get('qr_uri'))
+    print("If you are having trouble scanning the QR code, you can also add "
+          "the following code manually to your authenticator app: "
+          f"{json_data.get('otp_secret')}")
```
[the remark this answers is not in the document]
Comment (frankcorneliusmartin) [reply vantage6-common/vantage6/common/client/utils.py:23]: Stoute Bart

File: vantage6-server/vantage6/server/resource/collaboration.py [hunk 2]
```diff
@@ -70,28 +68,20 @@ def setup(api: Api, api_base: str, services: dict) -> None:
         CollaborationOrganization,
         path+'/<int:id>/organization',
         endpoint='collaboration_with_id_organization',
-        methods=('GET', 'POST', 'DELETE'),
```
Comment (frankcorneliusmartin) [inline vantage6-server/vantage6/server/resource/collaboration.py:73]: I assume this info can be retrieved elsewhere?

File: vantage6-server/vantage6/server/_data/node_config_skeleton.yaml
[the remark this answers is not in the document]
Comment (bartvanb) [reply vantage6-server/vantage6/server/_data/node_config_skeleton.yaml:1]: No idea where it is used. Shall we just create an issue to figure out if we still need this?

File: vantage6-server/vantage6/server/resource/collaboration.py (hunk 2 above)
Comment (bartvanb) [reply vantage6-server/vantage6/server/resource/collaboration.py:73]: You mean `/collaboration/<id>/organization`? Yes that is now `/organization?collaboration_id=<id>`

File: vantage6-server/vantage6/server/_data/node_config_skeleton.yaml
[the remark this answers is not in the document]
Comment (bartvanb) [reply vantage6-server/vantage6/server/_data/node_config_skeleton.yaml:1]: I removed this file and also the `server_config_skeleton.yaml` file next to it. Both were not used as far as I could tell.

**Pull request merged**